docs/terminology.rst

```diff
@@ -8,11 +8,11 @@ Terminology
     :header: "Name", "Description"
     :widths: 30, 60
 
-    **BuildSpec**,"``BuildSpec`` is a YAML file that buildtest interprets when generating the test. A BuildSpec may
-    contain one or more test blocks that is validated by a ``BuildSpec Schema``."
-    **BuildSpec Schema**,"is a JSON file defining  valid/invalid key value pairs and formatting for the file"
-    **Schema Library**,"A Schema Library is a collection of one or more ``BuildSpec Schema`` provided by buildtest."
-    **Test Script**,"A Test Script is a generated shell script by buildtest as a result of processing one of the BuildSpec."
+    **Buildspec**,"``Buildspec`` is a YAML file that buildtest interprets when generating the test. A Buildspec may
```

Comment:
nit: we don't need to say it twice, just do like "Buildspec: is a yaml file..." to follow convention of the others.

Reply:
got it. I will fix this and all the others as needed.

Reply:
Okay this is now fixed in latest commit.